java/core/src/test/java/co/worklytics/psoxy/rules/generics/CalendarTest.java

```diff
@@ -53,19 +34,19 @@ public void focusTimeBlockTitleSnippets_noMatch(String input) {
 
     @CsvSource(value = {
         "OOO,OOO",
-        "OOO: Vacation,OOO: ",
-        "OOO Conference,OOO ",
+        "OOO: Vacation,OOO",
+        "OOO Conference,OOO",
         "Out of Office,Out of Office",
-        "Out of the Office: Vacation,Out of the Office: ",
-        "Focus Time,Focus Time",
-        "Secret Project Focus Time,Focus Time",
-        "Focus Time Block,Focus Time Block",
-        "Focus: Secret Project,Focus: ",
-        "No Meeting Wednesday,No Meeting ",
-        " No Meetings,No Meetings",
-        "Prep Time,Prep Time",
-        "Prep Customer Meeting,Prep ",
-        "Prep: Customer,Prep: ",
+        "Out of the Office: Vacation,Out of the Office",
+        "Focus Time,'Focus Time,Focus'",
```

Comment:
so, this is reality of this; bc transform products ALL matches, not just first. In effect, it's more like categorization than redaction.

behavior is needed if you want "Team weekly" to match both "team" and "weekly" cases. that will end up as `team,weekly`.